### ISSUE TITLE
Air bronchogram

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6683]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
An air bronchogram is a pattern of air-filled (low-attenuation) bronchi on a background of opaque (high-attenuation) airless lung. The sign implies (a) patency of proximal airways and (b) evacuation of alveolar air by means of absorption (atelectasis) or replacement (eg, pneumonia) or a combination of these processes. In rare cases, the displacement of air is the result of marked interstitial expansion (eg, lymphoma).
PMID:18195376

Comment
See Figure 2 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722. 

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
